src/solver/simulation/include/antares/solver/simulation/simulation-runner.h

```diff
@@ -22,17 +21,33 @@
 
 #pragma once
 
-#include "antares/infoCollection/StudyInfoCollector.h"
-#include "antares/solver/misc/options.h"
-#include "antares/solver/simulation/ISimulationObserver.h"
-#include "antares/writer/i_writer.h"
+#include "antares/solver/simulation/adequacy.h"
+#include "antares/solver/simulation/economy.h"
```

Comment:
These headers are not required here, please move them to **simulation-runner.cpp**

Reply:
ok, they will have to be moved to **simulation-runner.cpp**, where they are needed.

Reply:
Done